build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+import org.gradle.internal.os.OperatingSystem
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    java
+    kotlin("jvm")
+}
+
+group = "com.fasterxml.jackson.module"
+version = "2.13.1-SNAPSHOT"
+
+dependencies {
+    implementation(kotlin("reflect"))
+    implementation(platform("com.fasterxml.jackson:jackson-bom:$version"))
+    implementation("com.fasterxml.jackson.core:jackson-databind")
+    implementation("com.fasterxml.jackson.core:jackson-annotations")
+
+    testImplementation("junit:junit:4.13.2")
+    testImplementation(kotlin("test-junit"))
+    testImplementation("com.fasterxml.jackson.dataformat:jackson-dataformat-xml")
+}
+
+val generatedSources = file("target/generated-sources")
+sourceSets {
+    main {
+        java.srcDirs(generatedSources)
+    }
+}
+
+val compatibilityTestEnabled =
+    providers
+        .gradleProperty("compatibilityTest")
+        .forUseAtConfigurationTime() // TODO: Remove `forUseAtConfigurationTime` when updating to Gradle 3.4
```

Comment:
Aren't we using Gradle 7.3.3? (maybe a copypasta error?)

Reply:
This should be:

```suggestion
        .forUseAtConfigurationTime() // TODO: Remove `forUseAtConfigurationTime` when updating to Gradle 7.4
```